tools/create-cluster

```diff
@@ -41,14 +43,17 @@ networking:
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
-  machineCIDR: 192.168.126.0/24
+  machineNetwork:
+  - cidr: 192.168.126.0/24
   networkType: OpenShiftSDN
   serviceNetwork:
   - 172.30.0.0/16
 platform:
   libvirt:
+    URI: qemu+tcp://192.168.122.1/system
```

Comment:
Is't it default or we need to explicit define it?

Reply:
yea, we can leave that out, i was copying line for line the generated install-config